src/wilcoxon.jl

```diff
@@ -129,12 +129,15 @@ function pvalue(x::ExactSignedRankTest; tail=:both)
     end
 end
 
+ci(x::ExactSignedRankTest, alpha::Real=0.05; tail=:both) = calculate_ci(x.vals, alpha; tail=tail)
+
 
 ## APPROXIMATE SIGNED RANK TEST
 
 immutable ApproximateSignedRankTest{T<:Real} <: HypothesisTest
+    vals::AbstractVector{T} # original values
     W::Float64              # test statistic: Wilcoxon rank-sum statistic
-    ranks::Vector{T}        # ranks without ties (zero values)
+    ranks::Vector{Float64} # ranks without ties (zero values)
```

Comment:
this always just returns `Float64` anyways so to prevent the weird coverage errors I ended up just explicitly defining this type, let me know if anyone has any issues with this